src/libslic3r/GCode.cpp

```diff
@@ -3942,65 +3942,134 @@ std::string GCode::_before_extrude(const ExtrusionPath &path, const std::string
         max_acceleration = m_config.machine_max_acceleration_extruding.get_at(0);
     double travel_acceleration = get_travel_acceleration(m_config);
     if(acceleration > 0){
+
+        if (m_config.travel_acceleration.value > 0)
+            travel_acceleration = m_config.travel_acceleration.get_abs_value(acceleration);
```

Comment:
prefer using the new get_travel_acceleration(m_config);
from 3 lines above, as it will compute the travel %from the machine travel limit if available.

Reply:
This was code from before merging. I think I can remove these lines completely.

Reply:
Using get_travel_acceleration now.